### DISCUSSION TITLE
posts/667

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#33] [Announcements]
# posts/667

안녕하세요, 카카오의 AI 모델 개발을 담당하는 카나나 알파(Kanana ⍺) 조...

https://tech.kakao.com/posts/667?fbclid=IwZXh0bgNhZW0CMTEAAR2KRMNXrRowcwwQskTckArkg8BGnDe9Yg-WI-QK8_vcgkHOKKrGK7c2ye4_aem_e94rOEMQ7wW7xY7QeeI7Rw

**Comment by youngjewoo** [comment]
멋져요

**Comment by wdkdot** [comment]
👍

**Comment by JihwanEom** [comment]
대단합니다
